src/ServiceControl.Audit/Infrastructure/WebApi/RootController.cs

```diff
@@ -65,7 +65,7 @@ public OkNegotiatedContentResult<object> Config()
                 },
                 Transport = new
                 {
-                    settings.TransportCustomizationType,
+                    settings.TransportType,
```

Comment:
Should we consider putting the transport name (or display name) in here? What about Persistence type/name/display name?

Reply:
We don't know them at this point - would require a new method. I've added a persistence type to it.